src/vochain/vochain.proto

```diff
@@ -111,6 +111,7 @@ message ProofArbo {
 	bytes siblings = 2;
 	bytes value = 3;
 	KeyType keyType = 4;
+	bytes weight = 5;
```

Comment:
I think we should make the difference more clear between `value` and `weight`. Maybe `leafWeight` and `votingWeight`?